### DISCUSSION TITLE
AVG

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#67] [General]
# AVG

Check if AVG is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/749ce49f-87b2-4ba0-bee9-b1d1df982367

**Comment by DogeBoi3544** [comment]
Note: This must be installed through the Windows Store, otherwise this app will not install.
